### PR TITLE
Update Custom Voice sample code

### PR DESCRIPTION
**Pull request by ForrestGumb**
## Purpose
1. Show instruction when dev didn't copy customvoice folder.
Some dev only copy [personal_voice_sample.py](https://github.com/Azure-Samples/cognitive-services-speech-sdk/blob/master/samples/custom-voice/python/personal_voice_sample.py), and encounter error on 'import customvoice'.
Show instruction. Tell them 'customvoice' should be copied together.

2. Reduce Personal Voice audio prompt file from 50 seconds to 5 seconds.
Also change the voice folder structure. After this change, consent wave will not be used as audio prompt file.

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[x] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

### CODE REVIEW

File: samples/custom-voice/python/personal_voice_sample.py [hunk 1]
```diff
@@ -9,7 +9,11 @@
 from time import sleep
 import os
 import logging
-import customvoice
+try:
+    import customvoice
+except ImportError:
+    print('Pleae copy folder https://github.com/ForrestGumb/cognitive-services-speech-sdk/tree/master/samples/custom-voice/python/customvoice and keep the same folder structure ask github.' )
```
Comment (yanchang-gyc) [inline samples/custom-voice/python/personal_voice_sample.py:15]: "ask github"? is it a typo?

File: samples/custom-voice/python/professional_voice_sample.py [hunk 2]
```diff
@@ -10,7 +10,11 @@
 import os
 import datetime
 import logging
-import customvoice
+try:
+    import customvoice
+except ImportError:
+    print('Pleae copy folder https://github.com/ForrestGumb/cognitive-services-speech-sdk/tree/master/samples/custom-voice/python/customvoice and keep the same folder structure ask github.' )
```
Comment (yanchang-gyc) [inline samples/custom-voice/python/professional_voice_sample.py:16]: same as above

File: samples/custom-voice/python/personal_voice_sample.py (hunk 1 above)
Comment (yulin-li) [reply samples/custom-voice/python/personal_voice_sample.py:15]: the link is not official

Comment (ForrestGumb) [reply samples/custom-voice/python/personal_voice_sample.py]: accepted

**Review by yanchang-gyc — APPROVED** [review]

**Review by yulin-li — APPROVED** [review]

**Pull request merged**